aresponses/main.py

```diff
@@ -90,7 +96,33 @@ def _host_matches(self, match_host):
                    (match_querystring and _text_matches_pattern(path_pattern, path_qs)):
                     path_matched = True
                     if _text_matches_pattern(method_pattern, method.lower()):
+                        if response is PassThrough:
+                            connector = TCPConnector()
+                            connector._resolve_host = partial(self._old_resolver_mock, connector)
+
+                            new_is_ssl = ClientRequest.is_ssl
+                            ClientRequest.is_ssl = self._old_is_ssl
+                            try:
+                                original_request = request.clone(
+                                    scheme='https' if request.headers['AResponsesIsSSL'] else 'http')
+
+                                headers = {k: v for k, v in request.headers.items() if k != 'AResponsesIsSSL'}
+
+                                async with aiohttp.ClientSession(connector=connector) as session:
+                                    async with getattr(session, method.lower())(
+                                        original_request.url, headers=headers,
+                                        data=(await request.read())
+                                    ) as r:
+                                        headers = {k: v for k, v in r.headers.items() if k.lower() == 'content-type'}
+                                        text = await r.text()
+                                        response = self.Response(
```

Comment:
why not just return the `r` response object instead of creating a new one?

Reply:
because `r` is a `ClientResponse` and we need to return a `web.Response`